.docker/parameters.yml

```diff
@@ -31,6 +31,9 @@ parameters:
     subject_rewrites: []
     example_rds_articles:
         '16846': https://repro.elifesciences.org/example.html
+    rds_articles:
```

Comment:
This format has been suggested and adopted, so I'll take it that it unblocks https://github.com/elifesciences/issues/issues/5347